src/Controls/src/Core/Region.cs

```diff
@@ -105,14 +105,12 @@ public Region Inflate(double left, double top, double right, double bottom)
 			{
 				var region = Regions[i];
 
-				if (i == 0) // this is the first line
-					region.Top -= top;
+				region.Top -= top;
 
 				region.Left -= left;
 				region.Width += right + left;
 
-				if (i == Regions.Count - 1) // This is the last line
-					region.Height += bottom + top;
+				region.Height += bottom + top;
 
 				rectangles[i] = region;
```

Comment:
Made these changes since moving the region up but not making it bigger only really makes sense if we can guarantee that the next line will be right below the top line. This doesn't work in cases where the top line is at the end of the line and goes a little onto the next line for example.